gnomad_qc/v4/resources/sample_qc_def_for_edit.json

```diff
@@ -0,0 +1,2256 @@
+                            "definition": "Number of singletons that are transversions."
+                        },
+                        "n_snp": {
+                            "definition": "Number of SNP alternate alleles."
+                        },
+                        "n_insertion": {
+                            "definition": "Number of insertion alternate alleles."
+                        },
+                        "n_deletion": {
+                            "definition": "Number of deletion alternate alleles."
+                        },
+                        "n_transition": {
+                            "definition": "Number of transition (A-G, C-T) alternate alleles."
+                        },
+                        "n_transversion": {
+                            "definition": "Number of transversion alternate alleles."
+                        },
+                        "n_star": {
+                            "definition": "Number of star (upstream deletion) alleles."
+                        },
+                        "r_ti_tv": {
+                            "definition": "Transition/Transversion ratio."
+                        },
+                        "r_ti_tv_singleton": {
+                            "definition": "Singleton Transition/Transversion ratio."
+                        },
+                        "r_het_hom_var": {
+                            "definition": "Het/HomVar call ratio."
+                        },
+                        "r_insertion_deletion": {
+                            "definition": "Insertion/Deletion allele ratio."
+                        },
+                        "bases_over_gq_threshold": {
+                            "definition": "Number of bases in the interval over passed GQ thresholds."
+                        },
+                        "bases_over_dp_threshold": {
+                            "definition": "Number of bases in the interval over passed DP thresholds."
+                        }
+                    }
+                }
+            }
+        },
+        "population_inference": {
+            "definition": "Struct containing genetic ancestry information assigned by applying a principal components analysis (PCA) on samples and using those PCs in a random forest classifier trained on known ancestry labels.",
+            "subfields": {
+                "training_pop": {
+                    "definition": "Sample's genetic ancestry group if used in training the Random Forest."
+                },
+                "pca_scores": {
+                    "definition": "Sample's scores for each genetic ancestry PC."
+                },
+                "pop": {
+                    "definition": "Sample's inferred genetic ancestry group label."
+                },
+                "prob_afr": {
+                    "definition": "Random forest probability that a sample is of African/African-American ancestry."
```

Comment:
I wonder if we should reword these all now to be "Random forest probability that a sample is in the African/African-American genetic ancestry group."? or something along those lines